src/types.ts

```diff
@@ -1,12 +1,19 @@
 import { Address } from '@celo/base/lib/address'
 
+export enum ResolutionKind {
```

Comment:
The pattern we're trying to use for enums is `CapitalizedCase`, in this case it'd be `Address` and `Nom`. (I can't find where we talked about it now, it was in a slack thread started by @jeanregisser )